packages/create-svelte/shared/+vitest+playwright+typescript/playwright.config.ts

```diff
@@ -0,0 +1,11 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+	webServer: {
+		command: 'npm run build && npm run preview',
+		port: 4173
+	},
+	testMatch: 'tests/**/*.{js,ts}'
```

Comment:
is there any reason not to just add this to the default `playwright.config.ts`? seems like a reasonable glob to use, and that way we don't need to maintain multiple variants